vnext/Microsoft.ReactNative/Views/ViewManagerBase.cpp

```diff
@@ -165,6 +165,24 @@ void ViewManagerBase::GetExportedCustomBubblingEventTypeConstants(
       // Keyboard events
       L"KeyUp",
       L"KeyDown",
+
+      // Pointer events
+      L"PointerCancel",
```

Comment:
Should we split up some of these changes into more granular diffs? For example, adding onPointer* events could be useful independent of Fabric.

Reply:
I agree it would be nice to have pointer events work in paper.  I didn't want to block my fabric work on waiting for paper to catch up. -- And I believe unless we run in bridgeless mode, the properties have to be on the viewmanager for fabric to be able to use them.  -- I believe this matches core where pointer events only work in fabric.  (At least that was the case when I last looked)

Certainly would be open to having pointer events on paper... and its likely something we'll want to do to help the transition be smooth.